mem/kv_mem.go

```diff
@@ -278,33 +313,42 @@ func (kv *memKV) put(
 			_, _ = kv.delete(suffix)
 		})
 	}
-	b, err := common.ToBytes(value)
-	if err != nil {
-		return nil, err
+	if !kv.noByte {
```

Comment:
remove all this code and suggestion on line 351

Reply:
But we need this to store correct data in the map. Basically no storing byte array when using interfaces and vice versa.